combobreaker.ino

```diff
@@ -58,9 +58,6 @@
 // Number of digits on a Master combo lock
 #define DIGITS 40
 
-// C's % is remainder instead of modulo
```

Comment:
The modulo operation is actually defined as being the remainder after division of one number by another—they are interchangeable nomenclature.